rpcs3/Emu/NP/np_helpers.cpp

```diff
@@ -62,6 +62,15 @@ namespace np
 		// npid->reserved[0] = 1;
 	}
 
+	void string_to_country_code(std::string_view str, SceNpCountryCode& country_code)
+	{
+		memset(&country_code, 0, sizeof(country_code));
+		for (int c = 0; c <= str.length(); c++)
```

Comment:
```suggestion
		ensure(str.size() == sizeof(SceNpCountryCode::data));
		std::memcpy(country_code.data, str.c_str(), sizeof(SceNpCountryCode::data));
```

Reply:
As i had corrected it before your suggestion appeared i tried manually apply it but get the following error:
![image](https://github.com/user-attachments/assets/4ce68817-c4da-412a-aa55-47ae84504ad5)

Reply:
Resolved by changing it from a `std::string_view` to a `std::string`.